src/main/java/org/cryptomator/ui/health/HealthCheckComponent.java

```diff
@@ -33,6 +34,9 @@ default Stage showHealthCheckWindow() {
 		@BindsInstance
 		Builder vault(@HealthCheckWindow Vault vault);
 
+		@BindsInstance
+		Builder windowToClose(@Nullable Stage window);
```

Comment:
Since "vault options" is the only way to start the workflow, this doesn't need to be nullable, does it?

Reply:
No, not now at least. It was more a thought for the future, but i guess YAGNI.

Reply:
Fixed in 03a7e89